assets/js/packages/floating-links/config.js

```diff
@@ -0,0 +1,158 @@
+			<path stroke="#667085" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M19.25 12a7.25 7.25 0 1 1-14.5 0 7.25 7.25 0 0 1 14.5 0Z"/>
+			<path stroke="#667085" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M15.25 12a3.25 3.25 0 1 1-6.5 0 3.25 3.25 0 0 1 6.5 0ZM7 17l2.5-2.5M17 17l-2.5-2.5m-5-5L7 7m7.5 2.5L17 7"/>
+		</svg>
+	`;
+
+	// Icon for Documentation link
+	const documentationIcon = `
+		<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+			<path stroke="#667085" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="m14.924 16.002.482 2.432c.106.537.682.895 1.286.8l1.64-.256c.604-.095 1.007-.607.9-1.145l-.481-2.431m-3.827.6-1.157-5.835c-.106-.538.297-1.05.9-1.145l1.64-.257c.605-.095 1.18.264 1.287.801l1.157 5.836m-3.827.6 3.827-.6M8.75 15.75v2.5a1 1 0 0 0 1 1h1.5a1 1 0 0 0 1-1v-2.5m-3.5 0v-8a1 1 0 0 1 1-1h1.5a1 1 0 0 1 1 1v8m-3.5 0h3.5m-7.5 0v2.5a1 1 0 0 0 1 1h1.5a1 1 0 0 0 1-1v-2.5m-3.5 0v-10a1 1 0 0 1 1-1h1.5a1 1 0 0 1 1 1v10m-3.5 0h3.5"/>
+		</svg>
+	`;
+
+	// Icon for Notifications link
+	const notificationsIcon = `
+		<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+			<path stroke="#667085" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M17.25 12v-2a5.25 5.25 0 1 0-10.5 0v2l-2 4.25h14.5l-2-4.25ZM9 16.75s0 2.5 3 2.5 3-2.5 3-2.5"/>
+		</svg>
+	`;
+
+	/**
+	 * Define links for the "free" version of the plugin
+	 */
+	const freeVersionLinks = [
+		{
+			title: __( 'Upgrade', 'business-directory-plugin' ),
+			icon: upgradeIcon,
+			url: navLinks.freeVersion.upgrade,
+			openInNewTab: true,
+		},
+		{
+			title: __( 'Support', 'business-directory-plugin' ),
+			icon: supportIcon,
+			url: navLinks.freeVersion.support,
+			openInNewTab: true,
+		},
+		{
+			title: __( 'Documentation', 'business-directory-plugin' ),
+			icon: documentationIcon,
+			url: navLinks.freeVersion.documentation,
+			openInNewTab: true,
+		},
+		{
+			title: __( 'Notifications', 'business-directory-plugin' ),
+			icon: notificationsIcon,
+			url: '#',
+			openInNewTab: false,
+			class: 's11-notifications-icon',
+		},
+	];
+
+	/**
+	 * Define links for the "pro" version of the plugin
+	 */
+	const proVersionLinks = [
+		{
+			title: __( 'Support & Docs', 'business-directory-plugin' ),
+			icon: supportIcon,
+			url: navLinks.proVersion.support_and_docs,
+			openInNewTab: true,
+		},
+		{
+			title: __( 'Notifications', 'business-directory-plugin' ),
+			icon: notificationsIcon,
+			url: '#',
+			openInNewTab: false,
+			class: 's11-notifications-icon',
+		},
+	];
+
+	/**
+	 * Define options
+	 */
+	const options = {
+		color: 'rgba(60,75,93,0.8)',
+		hoverColor: '#3C4B5D',
+		bgHoverColor: 'rgba(109,135,185,0.07)',
+		logoIcon: `
+		<svg width="35" height="46" viewBox="0 0 31 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Looks like we can cut down the svg size quite a bit with https://jakearchibald.github.io/svgomg/